docker/app
# remove references to status & render commands

**- What I did**
Remove references to status & render commands in README (app & examples)

**- How to verify it**
Check all README and verify there's no references to `docker app status` nor `docker app render` commands


**- A picture of a cute animal (not mandatory but encouraged)**

![image](https://user-images.githubusercontent.com/705411/66291252-a54bba80-e8e1-11e9-9a99-931d3a0ceaa9.png)
